communication/src/AsyncTlsConnection.cpp

```diff
@@ -248,6 +248,7 @@ void AsyncTlsConnection::write(std::shared_ptr<OutgoingMsg> msg) {
         write_msg_ = nullptr;
         write(write_queue_.pop());
       }));
+  LOG_DEBUG(logger_, "Write: " << KVLOG(peer_id_.value()));
```

Comment:
No trailing space is needed before KVLOG, otherwise, you will have 2 spaces.

Reply:
updated changes